test/functional/ui/float_spec.lua

```diff
@@ -433,6 +433,18 @@ describe('float window', function()
     eq(5, api.nvim_get_option_value('scroll', { win = float_win }))
   end)
 
+  it("lines('w$') is computed correctly after nvim_win_set_height with 'splitkeep=screen' #36056", function()
```

Comment:
all tests are assumed to test correctness.

```suggestion
  it("lines('w$') after nvim_win_set_height with 'splitkeep=screen' #36056", function()
```